tests/52-compat/__snapshots__/Container52CompatTest-should_throw_correct_exception_when_injecting_missing_class_in_the_constructor.snapshot-1

```diff
@@ -0,0 +1,5 @@
+  'Car'
```

Comment:
Here and elsewhere, why all this space and misaligned format?

Ideally, I would like something more along the lines of this:
```
  Car
    => Engine
    => UpperEngine
    => CombustionChamber: Error while making  'Valve': Class NonExisting1 does not exist
```

Reply:
Fixed in https://github.com/lucatume/di52/pull/14/commits/e623842ae44dbbcf77bd5115d14229a82ad962d4